pkg/client/client.go

```diff
@@ -0,0 +1,122 @@
+package client
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/jetstack/preflight/api"
+)
+
+// This variables are injected at build time.
```

Comment:
```suggestion
// These variables are injected at build time.
```

Reply:
Changed in a new commit (required signoff).